bigquery_etl/backfill/utils.py

```diff
@@ -6,25 +6,31 @@
 from typing import List
 
 import click
+from google.cloud import bigquery
+from google.cloud.exceptions import NotFound
 
 from bigquery_etl.backfill.parse import BACKFILL_FILE, Backfill
 from bigquery_etl.cli.utils import (
     paths_matching_name_pattern,
     qualified_table_name_matching,
 )
+from bigquery_etl.util import extract_from_query_path
 
+BACKFILL_DESTINATION_PROJECT = "moz-fx-data-shared-prod"
+BACKFILL_DESTINATION_DATASET = "backfills_staging_derived"
 
-def get_backfill_file_to_entries_map(
-    sql_dir, project_id, qualified_table_name
-) -> defaultdict[Path, List[Backfill]]:
-    """Return backfills with name pattern."""
+
+def get_qualifed_table_name_to_entries_map(
+    sql_dir, project_id, qualified_table_name, status=None
```

Comment:
Since you're using type hinting, maybe we should also have it added to function parameters? Like so:
`sql_dir: Path, project_id: str, qualified_table_name: str, status: str = None`

Reply:
I have added this to be considered in future iterations.